server/views/applications/pages/access-and-healthcare/access-needs.njk

```diff
@@ -0,0 +1,110 @@
+{% extends "../layout.njk" %}
+
+{% block questions %}
+
+  <h1 class="govuk-heading-l">{{page.title}}</h1>
+
+  {{ applyCheckboxes({
```

Comment:
[The Prototype](https://approved-premises-prototype-main.apps.live.cloud-platform.service.justice.gov.uk/access-needs) seems to show that we should be asking for details about these needs. It's in the screenshot (though with a yes/no), but not in here. Also the questions are in a slightly different order. I know you're discussing this with Beth, so /hides

Reply:
Yeah this confused me until I asked Beth.
[This is the latest version (v2)](https://approved-premises-prototype-main.apps.live.cloud-platform.service.justice.gov.uk/access-needs-v2). If you answer that the person has 'Mobility Needs' then you are taken to [this screen](https://approved-premises-prototype-main.apps.live.cloud-platform.service.justice.gov.uk/access-needs-follow-up) where you can add further details. 
We should probably remove that page that you linked from the prototype

Reply:
Ah ha! Perfect 👍